src/Analyzers/SolutionCodeFixApplier.cs

```diff
@@ -42,15 +45,25 @@ internal class SolutionCodeFixApplier : ICodeFixApplier
                 fixAllDiagnosticProvider: new DiagnosticProvider(result),
                 cancellationToken: cancellationToken);
 
-            var action = await fixAllProvider.GetFixAsync(fixAllContext);
-            var operations = await (action?.GetOperationsAsync(cancellationToken) ?? Task.FromResult(ImmutableArray<CodeActionOperation>.Empty));
-            var applyChangesOperation = operations.OfType<ApplyChangesOperation>().SingleOrDefault();
-            return applyChangesOperation?.ChangedSolution ?? solution;
+            try
+            {
+                var action = await fixAllProvider.GetFixAsync(fixAllContext).ConfigureAwait(false);
+                var operations = action is object
+                    ? await action.GetOperationsAsync(cancellationToken).ConfigureAwait(false)
+                    : ImmutableArray<CodeActionOperation>.Empty;
+                var applyChangesOperation = operations.OfType<ApplyChangesOperation>().SingleOrDefault();
+                return applyChangesOperation?.ChangedSolution ?? solution;
+            }
+            catch (Exception ex)
+            {
+                logger.LogWarning($"Failed to apply code fix {codeFix?.GetType().Name} for {diagnosticId}: {ex.Message}");
```

Comment:
localize this string

Reply:
I was planning to save the localization pass for when we knew which strings we wanted to keep. At the moment this is not a very helpful message, but I am not sure how to improve it. StyleCop fixers produce several of these failures.

Reply:
I don't see why the user wouldn't want to know that this is happening?

Reply:
Sure, it is just how to present it.